### PR TITLE
Stylistic changes

### PR DESCRIPTION
**Pull request by jarlsondre**
# Summary
Small stylistic changes to make code and tutorials more readable.

---

**Related issue :**

### CODE REVIEW

**Review by matbun — COMMENTED** [review]
Thank you for the fixes!
Since you brought up the problem of imports orderings, why not complying with some "standard"? You could run [isort](https://pycqa.github.io/isort/) on the whole package and *ideally* see if it can be activated in our [superlinter](https://github.com/super-linter/super-linter) CI/CD linter on GH. Alternativelly you can look for isort GH actions.

**Comment by jarlsondre** [comment]
@matbun I have now added support for isort and fixed the files that it complained about initially. It seems that when running `isort <filename>` from the command line without specifying anything else that it will not use the same style as the super-linter does. I could not seem to figure out which "profile" the super-linter uses for isort, but when adding "black" as the profile it seemed to work for the initial files that were tested. Therefore I added "black" as the profile in the `pyproject.toml` file. There are other profiles to choose from here, though: https://pycqa.github.io/isort/docs/configuration/profiles.html

**Review by matbun — APPROVED** [review]
LGTM

File: .isort.cfg
Comment (matbun) [inline .isort.cfg:1]: Could you see if this could be embedded into the `pyproject.toml` please? It would reduce cluttering of the repo's root

Comment (jarlsondre) [reply .isort.cfg:1]: Fixed this now :)

**Review by matbun — APPROVED** [review]

**Pull request merged**